portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,114 @@
+        <div class="collapse navbar-collapse flex-grow-1 text-right" id="myNavbar">
+            <ul class="navbar-nav ml-auto flex-nowrap">
+                <li class="nav-item">
+                    <a href="../portfolio.html" class="nav-link m-2 menu-item nav-active">Portfolio</a>
+                </li>
+                <li class="nav-item">
+                    <a href="../resume.html" class="nav-link m-2 menu-item">Resume</a>
+                </li>
+                <li class="nav-item">
+                    <a href="../about.html" class="nav-link m-2 menu-item">About</a>
+                </li>
+                <li class="nav-item">
+                    <a href="../contact.html" class="nav-link m-2 menu-item">Contact</a>
+                </li>
+            </ul>
+        </div>
+    </nav>
+
+    <!-- Main -->
+    <div class="container main-info-section about-section">
+        <div class="block">
+            <div class="card w-100  text-white bg-dark">
+            <div class="card-header">
+                About Me
+            </div>
+            <div class="card-body">
+                <p>I am a rising sophomore at Yale University.  I am interested in UX and front-end development.
+                My goal is to make accessible and intuitive technology for everyone but especially for elders.
+                I am majoring in computer science while also taking classes in psychology and visual arts. In my free time, I enjoy making music,
+                creating art, baking pastries, and trying new things.</p>
+                <footer ></footer> 
+            </div>
+            </div>
+
+            <div class="card w-100  text-white bg-dark" hidden>
+            <div class="card-header">
+                Mapping Out My Story 
+            </div>
+            <div class="card-body">
+                <div class="map-div">
+                        <div id="map"></div>
+                </div>  
+            </div>
+            </div>
+        </div>     
+    </div>
+
+    <!-- Footer -->
+    <footer class="footer font-small special-color-dark pt-4">
+        <div class="container">
+            <!-- Social buttons -->
+            <ul class="list-unstyled list-inline text-right">
+            <li> find me on: </li>
+            <li class="list-inline-item">
+                <a href="https://www.linkedin.com/in/lee-chen/" class="btn-floating btn-li mx-1">
+                <i class="fab fa-linkedin-in"> </i>
+                </a>
+            </li>
+            <li class="list-inline-item">
+                <a href="https://github.com/leeannachen" class="btn-floating btn-github mx-1">
+                <i class="fab fa-github"> </i>
+                </a>
+            </li>
+            </ul>
+        </div>
+    </footer>
+
+    <!-- key=AIzaSyDTkaBUiXmTof1JeMXiplYaeTZ39ms4DNU -->
+    <!-- <script async defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDTkaBUiXmTof1JeMXiplYaeTZ39ms4DNU&callback=initMap">
+    </script>  -->
```

Comment:
Should these be deleted if they are not needed? Also because you probably don't want keys stored in github.

Reply:
yup, it's quite messy right now because i am migrating everything in index.html to separate pages